tools/nnpackage_tool/model2nnpkg/model2nnpkg.py

```diff
@@ -0,0 +1,144 @@
+
+
+def _get_args():
+    parser = argparse.ArgumentParser(
+        description='Convert model files (tflite, circle or tvn) to nnpackage.',
+        usage=''' %(prog)s [options]
+  Examples:
+      %(prog)s -m add.tflite                           => create nnpackage "add" in current directory
+      %(prog)s -o out -m add.tflite                    => create nnpackage "add" in out/
+      %(prog)s -o out -p addpkg -m add.tflite          => create nnpackage "addpkg" in out/
+      %(prog)s -c add.cfg -m add.tflite                => create nnpackage "add" with add.cfg
+      %(prog)s -o out -p addpkg -m a1.tflite a2.tflite => create nnpackage "addpkg" with models a1.tflite and a2.tflite in out/
+  ''')
+    parser.add_argument(
+        '-o',
+        '--outdir',
+        type=str,
+        default=os.getcwd(),
+        metavar='output_directory',
+        help='set nnpackage output directory')
+    parser.add_argument(
+        '-p',
+        '--nnpackage-name',
+        type=str,
+        metavar='nnpackage_name',
+        help='set nnpackage output name (default=[1st modelfile name])')
+    parser.add_argument(
+        '-c',
+        '--config',
+        type=str,
+        nargs='+',
+        default='',
+        metavar='conf',
+        help='provide configuration files')
+    parser.add_argument(
+        '-m',
+        '--models',
+        type=str,
+        nargs='+',
+        metavar='model',
+        help='provide model files')
+
+    args = parser.parse_args()
+
+    _verify_args(args)
+
+    if not args.nnpackage_name:
+        first_model_name = os.path.basename(args.models[0]).rsplit('.', 1)[0]
+        args.nnpackage_name = first_model_name
+
+    args.prog = parser.prog
+
+    return args
+
+
+def _generate_manifest(args):
+    config_list = ''
+    if args.config:
+        config_list = [os.path.basename(e) for e in args.config]
+    models_list = [os.path.basename(e) for e in args.models]
+    types_list = [os.path.basename(e).rsplit('.', 1)[1] for e in args.models]
+    # types_str = ','.join("{}".format(os.path.basename(e).rsplit('.', 1)[1]) for e in args.models)
+
+    manifest = {}
+    manifest["major-version"] = "1"
+    manifest["minor-version"] = "2"
+    manifest["patch-version"] = "0"
+    manifest["configs"] = config_list
+    manifest["models"] = models_list
+    manifest["model-types"] = types_list
+
+    return manifest
+
+
+def main():
+    # parse arguments
+    args = _get_args()
+    print(args)
+
+    print('{}: Generating nnpackage {} in {}'.format(args.prog, args.nnpackage_name,
+                                                     args.outdir))
+    # mkdir nnpackage directory
+    nnpackage_path = os.path.join(args.outdir, args.nnpackage_name)
+    os.makedirs(os.path.join(nnpackage_path, 'metadata'), exist_ok=True)
+
+    # dump menifest file
+    manifest = _generate_manifest(args)
+    menifest_path = os.path.join(nnpackage_path, 'metadata', 'MANIFEST')
+    with open(menifest_path, "w") as json_file:
```

Comment:
Please search and replace all `menifest_path`
```suggestion
    with open(manifest_path, "w") as json_file:
```